cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AliasesTest.scala

```diff
@@ -192,23 +212,72 @@ class AliasesTest extends DocumentingTest with QueryStatisticsTestSupport {
           statsOnlyResultTable()
         }
         p("When a database alias pointing to a remote database has been created, its details can be shown with the `SHOW ALIASES FOR DATABASE` command.")
-        query("SHOW ALIASES FOR DATABASE WHERE name = 'remote-northwind'", assertNameField("remote-northwind")) {
+        query("SHOW ALIAS `remote-northwind` FOR DATABASE", assertNameField("remote-northwind")) {
           resultTable()
         }
-        query("SHOW ALIASES FOR DATABASE YIELD * WHERE name = 'remote-with-driver-settings'", assertNameField("remote-with-driver-settings")) {
+        query("SHOW ALIAS `remote-with-driver-settings` FOR DATABASE YIELD *", assertNameField("remote-with-driver-settings")) {
           resultTable()
         }
+        p("Just as the local aliases, the remote database aliases can be given properties.")
+        query(
+          """CREATE ALIAS `remote-northwind-2021`
+            |FOR DATABASE `northwind-graph-2021` AT 'neo4j+s://location:7687'
+            |USER alice PASSWORD 'password'
+            |PROPERTIES { newestNorthwind: false, index: 6 }""".stripMargin,
+          ResultAssertions(r => { assertStats(r, systemUpdates = 1) })
+        ) { statsOnlyResultTable() }
+        p("The properties are then shown in the `SHOW ALIASES FOR DATABASE YIELD ...` command.")
+        query("SHOW ALIAS `remote-northwind-2021` FOR DATABASE YIELD name, properties", assertNameField("remote-northwind-2021")) { resultTable() }
         p(
           """Creating remote aliases also allows `IF NOT EXISTS` and `OR REPLACE` clauses.
             |Both check for any remote or local database aliases.""".stripMargin)
       }
+
+      section(title = "Create database aliases in composite databases", id = "database-management-create-database-alias-in-composite", role = "enterprise-edition") {
+        p(
+          """Both local and remote database aliases can be part of <<cypher-manual#administration-databases-create-composite-database, composite databases>>.
+            |Creating a database alias in a composite database is done by giving the composite database name as namespace for the alias name.""".stripMargin)
```

Comment:
Line 239 is a little bit confusing. Did you mean something like this?

```suggestion
            |You can create a database alias in a composite database by naming the composite database as the alias name.""".stripMargin)
```

Reply:
I guess, not sure how much we use `You` though...

Reply:
It's actually advisable to avoid it, but the problem is with using "we". Maybe one way to find a middle term would be something like 
```suggestion
            |Create a database alias in a composite database by naming the composite database with the same alias name.""".stripMargin)
```

Reply:
what about:

```suggestion
            |Create a database alias in a composite database by giving the composite database name as namespace for the alias name.""".stripMargin)
```

Reply:
and you are not naming the alias after the composite... you are giving the composite as a namespace:
Composite db: comp
Alias name: alias1
CREATE ALIAS comp.alias1 FOR ...

Reply:
Trying to avoid too much repetition of "name" so it doesn't get as confusing 😅 

```suggestion
            |Create a database alias in a composite database by giving the name of the composite database as namespace for the alias.""".stripMargin)
```